test/models/developers/finder_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class Developers::FinderTest < ActiveSupport::TestCase
```

Comment:
We need tests for when the feature is disabled, too. Right now we are only testing two of the three possibilities.

We can mock the feature with the following:

```ruby
Feature.stub(:enabled?, true) do
  # Do the stuff.
  # Feature.enabled? will be reset at the end of the block.
end
```

We should make sure to stub

Reply:
Added the tests with the feature being disabled!